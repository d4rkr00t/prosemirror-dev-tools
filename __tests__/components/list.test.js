import React from "react";
import renderer from "react-test-renderer";

jest.dontMock("../../src/components/list");

test.only("ListItemGroup renders when clicked", () => {
  const module = require("../../src/components/list");
  const component = module.List({
    title: jest.fn(),
    groupTitle: jest.fn(),
    items: ["", "", ["", "", "", ""]]
  });
  let tree = renderer.create(component).toJSON();
  expect(tree).toMatchSnapshot();
});
