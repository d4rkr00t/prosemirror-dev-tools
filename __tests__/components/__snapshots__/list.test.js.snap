// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListItemGroup renders when clicked 1`] = `
<div>
  <button
    className="sc-bdVaJa bXvpxj"
    onClick={[Function]}
    onDoubleClick={[Function]}
  />
  <button
    className="sc-bdVaJa bXvpxj"
    onClick={[Function]}
    onDoubleClick={[Function]}
  />
  <div>
    <button
      className="sc-bdVaJa bXvpxj"
      onClick={[Function]}
    >
      <div
        style={
          Object {
            "flexGrow": 1,
          }
        }
      />
      <div>
        â–¶
      </div>
    </button>
    <div
      className="sc-bwzfXH dlTaFc"
    >
      <button
        className="sc-bdVaJa biMaei"
        onClick={[Function]}
        onDoubleClick={[Function]}
      />
      <button
        className="sc-bdVaJa biMaei"
        onClick={[Function]}
        onDoubleClick={[Function]}
      />
      <button
        className="sc-bdVaJa biMaei"
        onClick={[Function]}
        onDoubleClick={[Function]}
      />
      <button
        className="sc-bdVaJa biMaei"
        onClick={[Function]}
        onDoubleClick={[Function]}
      />
    </div>
  </div>
</div>
`;
